package.json

```diff
@@ -23,20 +25,27 @@
     "url": "https://github.com/NexusMutual/static-data/issues"
   },
   "homepage": "https://github.com/NexusMutual/static-data#readme",
+  "engines": {
+    "node": ">=10.4.0"
+  },
   "dependencies": {
     "@nexusmutual/deployments": "^2.4.1",
+    "axios": "^1.6.8",
     "ethers": "^5.7.2",
```

Comment:
I agree, we should choose either one. I rather use BigInt so either ethers v6 or viem.

we should also consider what ether-fi or other partners are mostly using for easier integration

Reply:
If we switch I would prefer ethers since is more stable from what I heard on the dailies, but again not sure how many projects switched to this pck, so I think string is still the safest one to put in here...

Reply:
let's create an issue and decide later on, uniformly, for all backend repos

Reply:
https://github.com/NexusMutual/sdk/issues/115